sdk/ahnlich-client-py/README.md

```diff
@@ -0,0 +1,386 @@
+## Ahnlich Client PY
+
+A Python client that interacts with both ahnlich DB and AI
+
+
+[![Ahnlich TestSuite](https://github.com/deven96/ahnlich/actions/workflows/test.yml/badge.svg)](https://github.com/deven96/ahnlich/actions/workflows/test.yml)
+[![Ahnlich Python Client Tag and Deploy](https://github.com/deven96/ahnlich/actions/workflows/python_tag_and_deploy.yml/badge.svg)](https://github.com/deven96/ahnlich/actions/workflows/python_tag_and_deploy.yml)
+
+## Usage Overview
+
+The following topics are covered:
+* [Installation](#installation)
+* [Package Information](#package-information)
+* [Server Response](#server-response)
+* [Initialization](#initialization)
+    * [Protocol](#protocol)
+    * [Client](#client)
+
+* [Requests](#requests)
+    * [Ping](#ping)
+    * [Info Server](#info-server)
+    * [List Connected Clients](#list-connected-clients)
+    * [List Stores](#list-stores)
+    * [Create Store](#create-store)
+    * [Drop Store](#drop-store)
+    * [Get Key](#get-key)
+    * [Get By Predicate](#get-by-predicate)
+    * [Create Index](#create-index)
+    * [Drop Index](#drop-index)
+    * [Set](#set)
+    * [Delete Key](#delete-key)
+    * [Delete Predicate](#delete-predicate)
+* [Bulk Requests](#bulk-requests)
+* [How to Deploy to Artifactory](#deploy-to-artifactory)
+* [Type Meanings](#type-meanings)
+* [Change Log](#change-log)
+
+## Installation
+
+- Using Poetry
+```bash
+poetry add ahnlich-client-py
+```
+- Using pip
+```bash
+pip3 install ahnlich-client-py
+```
+
+## Package Information
+The ahnlich client has some noteworthy modules that should provide some context
+- Bincode
+- Serde Types
+- Serde Binary
+
+The above mentioned are classes generated by `serde_generate` to help represent the primitive rust types and provide a base bincode serialization capabilities
+
+- Query: Generated from the spec document, contains all the types used by to send a request to the ahnlich database
+- Server Response: Generated from the spec document, contains all  the possible server response.
+- Builders:
+- Exceptions: Possible Client Exceptions
+- Libs: Contains helpers, such as `create_store_key`
+
+
+## Server Response
+
+All query types have an associating server response, all which can be found
+```py
+from ahnlich_client_py import server_response
+```
+
+## Initialization
+
+### Protocol
```

Comment:
Not sure we have to show the protocol being initialized as it's own subheader as it is always used in the context of a client